www/js/app.js

```diff
@@ -513,8 +513,7 @@ define(['jquery', 'zimArchiveLoader', 'uiUtil', 'settingsStore','abstractFilesys
                 // is indeed unregistered but still active...
                 // So we have to disable it manually (even if it's still registered and active)
                 navigator.serviceWorker.controller.postMessage({'action': 'disable'});
-                messageChannel = null;
```

Comment:
What is the rationale for removing this? We are trying to disable the messageChannel. Have you determined that this method is ineffective?

Reply:
actually in the codefactor it  is considered as an unused element .. so i thought that it is something which is not in use . so i removed it

Reply:
OK, no I think we need to keep this (as you have now done).